src/Pie.js

```diff
@@ -18,37 +18,71 @@ function createPath(cx, cy, r, startAngle, arcAngle) {
       r,
       startAngle,
       startAngle + arcAngle,
-    )
+    );
   } else {
-    p.path.push(0, cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle))
-    p.path.push(4, cx, cy, r, startAngle, startAngle + arcAngle, 1)
+    p.path.push(0, cx + r * Math.cos(startAngle), cy + r * Math.sin(startAngle));
+    p.path.push(4, cx, cy, r, startAngle, startAngle + arcAngle, 1);
   }
-  return p
+  return p;
 }
 
-const ArcShape = ({ radius, width, color, strokeCap,startAngle, arcAngle }) => {
+const ArcShape = ({ radius, width, color, strokeCap, startAngle, arcAngle }) => {
   const path = createPath(
     radius,
     radius,
     radius - width / 2,
     startAngle / 180 * Math.PI,
-    arcAngle / 180 * Math.PI
-  )
-  return <Shape d={path} stroke={color} strokeWidth={width} strokeCap={strokeCap} />
-}
+    arcAngle / 180 * Math.PI,
+  );
+  return <Shape d={path} stroke={color} strokeWidth={width} strokeCap={strokeCap} />;
+};
 
-const RingShape = props =>
-  Platform.OS === 'ios'
-    ? <ArcShape {...props} startAngle={0} arcAngle={360} />
-    : <Group>
-        <ArcShape {...props} startAngle={0} arcAngle={180} />
-        <ArcShape {...props} startAngle={180} arcAngle={180} />
-      </Group>
+const RoundDividers = ({ paintedSections, dividerSize, width, radius, backgroundColor }) => {
+  let dividerColorOverlayArray = [];
+  let dividerArray = [];
+  paintedSections.forEach((section, index) => {
+    const { percentage, color, startAngle } = section;
+    const shouldShow = percentage !== 100;
+
+    dividerArray.push(shouldShow && (<ArcShape
+      key={index}
+      radius={radius}
+      width={width}
+      color={backgroundColor}
+      startAngle={startAngle - dividerSize / 2}
+      arcAngle={dividerSize}
+      strokeCap={'round'}
+    />));
+
+    dividerColorOverlayArray.push(shouldShow && (<ArcShape
+      key={index}
+      radius={radius}
+      width={width}
+      color={color}
+      startAngle={startAngle + section.arcAngle - dividerSize / 2 - 1}
+      arcAngle={1}
+      strokeCap={'round'}
+    />));
+
+
+  });
+  return (
+    <Group>
+      {dividerArray}
+      {dividerColorOverlayArray}
+    </Group>
+  );
+};
+
+const getArcAngle = (percentage) => percentage / 100 * 360;
+
+const Pie = ({ sections, radius, innerRadius, backgroundColor, strokeCap, dividerSize }) => {
+  const width = radius - innerRadius;
+  const backgroundPath = createPath(radius, radius, radius - width / 2, 0, 360);
+  const shouldShowRoundDividers = !!dividerSize && strokeCap === 'round';
```

Comment:
```js
const hasValidPercentage = sections.every(({ percentage }) => percentage !== 100);
const shouldShowRoundDividers = !!dividerSize && strokeCap === 'round' && hasValidPercentage;
```

Reply:
As discussed in a different thread, we won't do this here